tests/README.md

```diff
@@ -1,10 +1,29 @@
 # VIC Engine Integration & Functional Test Suite
 
-To run the integration tests locally:
+Integration tests can be run locally in several different ways.
 
 ## Automatic with defaults
 
-Use ./local-integration-test.sh
+The recommended way to run integration tests is [local-integration-test.sh][sh].
+
+This script requires that `govc` is [installed][govc-install] and available on
```

Comment:
In part because the script provides a sane error, I chose not to document that requirement here. (This is already wordy enough, and I hope that someone following these instructions will download the latest version.)